stress/common/src/main/java/alluxio/stress/worker/UfsIOParameters.java

```diff
@@ -31,10 +35,16 @@
   public String mDataSize = "4G";
 
   @Parameter(names = {"--path"},
-          description = "the Alluxio directory to write temporary data in",
+          description = "the Ufs Path to write temporary data in",
           required = true)
   public String mPath;
 
+  @Parameter(names = {"--use-ufs-conf"},
+      description = "If true, use the existing ufs configuration to read/write to the base path")
+  @Parameters.BooleanDescription(trueDescription = "UseUFSConf",
```

Comment:
I'm not familiar with this. What does this do?

Reply:
not sure? i modeled after other parameters, but removed for now for simplicity.

Reply:
O yeah, I remember this now. This is for graphing later. It is useful to have a description for true and false, instead of just displaying `true` or `false`. This probably won't matter in this case.